onnxruntime/core/session/inference_session.cc

```diff
@@ -56,11 +56,419 @@
 #include "core/session/custom_ops.h"
 #endif
```

Comment:
Should keep in a new branch for this feature, this PR is just for quick code sharing